src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -5208,8 +5208,24 @@ def _build_arguments_schema(cls, *args, **kwargs):
             template="/subscriptions/{subscription}/resourceGroups/{resource_group}/providers/Microsoft.Network"
                      "/ddosProtectionPlans/{}",
         )
+        args_schema.ip_tags = AAZDictArg(
+            options=["--ip-tags"],
+            help="Space-separated list of IP tags in `TYPE=VAL` format.",
+            nullable=True
+        )
+        args_schema.ip_tags.Element = AAZStrArg(nullable=True)
```

Comment:
If you set the element is nullable, then the implementation below should be modified.